keras/src/wrappers/sklearn_wrapper.py

```diff
@@ -13,13 +13,15 @@
 from keras.src.wrappers.utils import assert_sklearn_installed
 
 try:
-    import sklearn
     from sklearn.base import BaseEstimator
     from sklearn.base import ClassifierMixin
     from sklearn.base import RegressorMixin
     from sklearn.base import TransformerMixin
+    from sklearn.pipeline import make_pipeline
+    from sklearn.preprocessing import OneHotEncoder
+    from sklearn.utils.metadata_routing import MetadataRequest
+    from sklearn.utils.validation import check_is_fitted
 except ImportError:
```

Comment:
It seems to be there's a possibility for `ImportError` to be raised even if sklearn is installed, if there's a version mismatch. This could confuse users.